src/sentry/api/endpoints/release_threshold.py

```diff
@@ -64,3 +72,30 @@ def post(self, request: Request, project: Project) -> HttpResponse:
             environment=result.get("environment"),
         )
         return Response(serialize(release_threshold, request.user), status=201)
+
+    def get(self, request: Request, project: Project) -> HttpResponse:
+        request.data["project"] = project.slug
```

Comment:
If we have the project already from the path params, do we need to pul the slug from the request data?
The class mixin already converts the slug into the project instance, so it should be the same i believe?

Reply:
This is just validating the project - since it doesn't come in in the request data I stick the slug into it so we can validate that way, is this already being done in some middleware (i.e can we skip it here?)

Reply:
Since we're inheriting from the project mixin i imagine that it is?

Reply:
i think this is already being validated in `ProjectEndpoint`